jscomp/test/UncurriedExternals.js

```diff
@@ -36,22 +34,20 @@ let te = {
 let tcr = {};
 
 function tsiC(c) {
-  c.increment = (function (amount) {
+  c.increment = function (amount) {
     let me = this ;
     console.log(me);
-  });
+  };
 }
 
 function tsiU(c) {
-  c.increment = (function (amount) {
+  c.increment = function (amount) {
```

Comment:
Here is a non-top-level function that does not use an arrow yet.

Reply:
Because there is a `this` binding below

Reply:
Ah, ok!